README.md

```diff
@@ -105,27 +106,27 @@ When running Download Directory in Jenkins, the code is cloned from Github each
 
 <h3> Verifying Download Directory Results</h3>
 
 This section will touch on the <b>files</b> produced by each step in Download Directory, and how to verify they were produced correctly. Often, comparing the files produced in the previous release is the way to go, but where needed this guide will provide additional suggestions for checking the output.
 
-<b>Note</b>: During Download Directory, the output files are temporarily held in a folder corresponding to the current release. For release 67, the output files would appear in `data-release-pipeline/downloadDirectory/67/`. 
+<b>Note</b>: During Download Directory, the output files are temporarily held in a folder corresponding to the current release. For release 67, the output files would appear in `release-download-directory/download-directory/67/`. 
 
 <h4>DatabaseDumps</h4>
 
-This step generates two mySQL dump files from the `stable_identifiers` and `test_reactome` databases. The files produced are `gk_stable_ids.sql.gz` and `gk_current.sql.gz`, respectively. These are then placed in the `data-release-pipeline/downloadDirectory/67/databases/` folder (if it was release 67). Comparing the size of these files to the previous release is sufficient for verifying the success of this step. 
+This step generates two mySQL dump files from the `stable_identifiers` and `release_current` databases. The files produced are `gk_stable_ids.sql.gz` and `gk_current.sql.gz`, respectively. These are then placed in the `release-download-directory/download-directory/67/databases/` folder (if it was release 67). Comparing the size of these files to the previous release is sufficient for verifying the success of this step. 
 
 <h4>BioPAX</h4>
 
-This step generates <b>BioPAX level 2</b> and <b>level 3</b> files for each species in the `test_reactome` database. Additional information on BioPAX can be found at its <a href="http://www.biopax.org/">website</a>. This will typically be the longest running step of Download Directory, particularly due to the generation of the level 3 BioPAX files. It makes use of the Pathway-Exchange jar that should have been locally installed during the <a href="https://github.com/reactome/data-release-pipeline/tree/feature/download-directory/downloadDirectory#-preparing-and-running-download-directory">preparation</a> step of Download Directory.
+This step generates <b>BioPAX level 2</b> and <b>level 3</b> files for each species in the `release_current` database. Additional information on BioPAX can be found at its <a href="http://www.biopax.org/">website</a>. This will typically be the longest running step of Download Directory, particularly due to the generation of the level 3 BioPAX files. It makes use of the Pathway-Exchange jar that should have been locally installed during the <a href="#-preparing-and-running-download-directory">preparation</a> step of Download Directory.
```

Comment:
The line `This will typically be the longest running step of Download Directory, particularly due to the generation of the level 3 BioPAX files` is outdated. BioPAX does tend to take a little longer then other steps, but everything now takes less than an hour, down from first 24 hours and then 13-15 hours. No more bottlenecks!

Reply:
Okay, that line is removed: https://github.com/reactome/release-download-directory/pull/2/commits/15cbc8ebc0eb71c87902b00bbb15a6923db07bdd